orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_api.xml.j2

```diff
@@ -189,7 +189,7 @@
             {% if 'OOZIE' in exposed and 'OOZIE_SERVER' in salt['pillar.get']('gateway:location') -%}
             <param>
                 <name>OOZIE</name>
-                <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000</value>
+                <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000;enableStickySession=false;noFallback=false;enableLoadBalancing=true</value>
```

Comment:
No. This should have been done in the initial change itself for Knox to Oozie LB. I had to fold it here because I had to test this patch in 5 different network configurations across AWS and Azure.